android/build.gradle

```diff
@@ -16,6 +16,8 @@ android {
 }
 
 dependencies {
-    implementation 'com.usabilla.sdk:ubform:6.+'
+    implementation 'com.usabilla.sdk:ubform:7.+'
     implementation "com.facebook.react:react-native:0.+"
+    implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
```

Comment:
Where is this coming from?

Reply:
After migrating to android x, it was getting crashed for swiperefreshlayout thats why i added this.

Reply:
Removed it as with new build its working, will check the sample app may be we have to update the sample app RN version.